.github/workflows/web-lint.yml

```
@@ -0,0 +1,109 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  web_lint:
+    name: web_lint
+    if: ${{ github.event_name != 'pull_request' || github.event.pull_request.draft == false }}
+    runs-on: ["self-hosted", "ragflow-test"]
+    steps:
+      - name: Ensure workspace ownership
+        run: echo "chown -R ${USER} ${GITHUB_WORKSPACE}" && sudo chown -R ${USER} ${GITHUB_WORKSPACE}
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:62: Double quote to prevent globbing and word splitting

(shellcheck)